src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__link {
+  display: block;
+  color: black;
+  line-height: 60px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+
+  transition: 0.2s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Optional:

It is a great practice to create variables for the colors/fonts you use in your code. 
Imagine the page was redesigned - what will you do? You will need to fix all the colors/fonts manually. But if you use a variable - you can just change it in a single place